packages/webpack-plugin/src/index.js

```diff
@@ -19,14 +19,15 @@ class LoadablePlugin {
 
   handleEmit = compilation => {
     const stats = compilation.getStats().toJson({
-      hash: true,
-      publicPath: true,
+      all: false,
       assets: true,
       chunks: true,
-      modules: false,
-      source: false,
-      errorDetails: false,
-      timings: false,
+      chunkGroups: true,
```

Comment:
Updated to:
- remove the falsey options (not needed now that we're specifying all: false)
- added `chunkGroupChildren: true` and `ids: true`.  These are necessary for webpack 5 otherwise https://github.com/gregberge/loadable-components/blob/cba6cab47536e23ded8bac34ee24417e6452344e/packages/server/src/ChunkExtractor.js#L273 and https://github.com/gregberge/loadable-components/blob/main/packages/server/src/ChunkExtractor.js#L298 will fail